### PR TITLE
dev-php/scrypt: new ebuild

### PR DESCRIPTION
**Pull request by SpiderX**
This is a PHP library providing a wrapper
to Colin Percival's scrypt implementation.
Scrypt is a key derivation function designed
to be far more secure against hardware brute-force
attacks than alternative functions
such as PBKDF2 or bcrypt.

Signed-off-by: Vladimir Pavljuchenkov <spiderx@spiderx.dp.ua>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: dev-php/scrypt/scrypt-1.4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PHP_EXT_ECONF_ARGS="--enable-scrypt"
+USE_PHP="php5-6 php7-0 php7-1 php7-2"
```
Comment (hydrapolic) [inline dev-php/scrypt/scrypt-1.4.2.ebuild]: Does it work with PHP 7.3?

Comment (SpiderX) [reply dev-php/scrypt/scrypt-1.4.2.ebuild]: Yes, it does.
Added php7-3 target.

**Comment by SpiderX** [comment]
@hydrapolic Is it ready for merge? or something still needs to be done?

**Comment by hydrapolic** [comment]
I'm just a stranger commenting on PR, the php/proxy maint people will have to review and merge it.

**Comment by AndrewAmmerlaan** [comment]
Hi there,

Sometimes 'new package' PRs get snowed under by all the other ones (new packages are low priority compared to fixes to already existing packages). If you'd like to make this ebuild available to the Gentoo users anyway, might I suggest the [GURU overlay](https://wiki.gentoo.org/wiki/Project:GURU). This is an ebuild repository maintained collaboratively by Gentoo users. To gain access to the repository, simply create an "Access request" on bugs.gentoo.org.

File: dev-php/scrypt/scrypt-1.4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PHP_EXT_ECONF_ARGS="--enable-scrypt"
+USE_PHP="php7-2 php7-3 php7-4"
```
Comment (juippis) [inline dev-php/scrypt/scrypt-1.4.2.ebuild:7]: Could you update this to reflect today? Ie, php-7.2 is masked and getting removed soon, and there's php-8.0 available.

Comment (SpiderX) [reply dev-php/scrypt/scrypt-1.4.2.ebuild]: Updated.
1. PHP 7.2 removed
2. PHP 8.0 added with patch from upstream's master.

**Review by juippis — APPROVED** [review]

**Comment by hydrapolic** [comment]
Happy to see it got merged @juippis.

**Pull request closed without merging**